website_docs/_index.md

```diff
@@ -12,16 +12,16 @@ export data.
 
 ## Status and Releases
 
-|       | Tracing | Metrics |
-| ----- | ------- | ------- |
-| API   | GA      | Alpha   |
-| SDK   | Beta    | Alpha   |
+| Signal  | API Status  | SDK Status        |
+|---------|-------------|-------------------|
+| Tracing | Stable      | Release Candidate |
```

Comment:
i believe we could use `trace` as signal name like done with the sdk names there: #2340

Reply:
Fixed:)